inlong-common/src/main/java/org/apache/inlong/common/heartbeat/ComponentHeartbeat.java

```diff
@@ -39,17 +43,38 @@ public class ComponentHeartbeat {
 
     private String inCharges;
 
+    // node load
+    private int loadValue;
```

Comment:
Suggest using `Integer`, its default value is `null`, which is suitable for usage.

And, personally, `value` is not needed in the param's name.

Reply:
Ok, let me fix it